forms-web-app/src/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  APPEALS_SERVICE_API_URL: process.env.APPEALS_SERVICE_API_URL,
```

Comment:
not sure how else to do this. Don't want references to `process.env` dotted everywhere we need an API call. There may be a better / more preferable way?

Reply:
Yeah, basically correct. See how I did it in [appeals-service-api](https://github.com/foundry4/appeal-planning-decision/blob/master/appeals-service-api/src/lib/config.js). Separating them into sections makes it more readable. Suggest...

```
module.exports = {
   appeals: {
     url: process.env.APPEALS_SERVICE_API_URL
   }
}
```

Reply:
done as suggested